### DISCUSSION TITLE
Unable to find Vue SFC Data ( State ) in devtools. Is this feature still WIP ?

### DISCUSSION THREAD

**Discussion by jaybharadia** [nuxt/devtools#245] [Q&A]
## **Screenshot** from **vue devtools** for reference 
<img width="1214" alt="image" src="https://github.com/nuxt/devtools/assets/28944638/4da5cf90-c6bd-4269-9035-967e5b28a397">

**Comment by arashsheyda** [comment]
we can track this on #244
